mlflow/azureml/__init__.py

```diff
@@ -28,7 +28,7 @@
 _logger = logging.getLogger(__name__)
 
 
-@experimental
+@deprecated(since="1.19.0")
```

Comment:
Can we add the `alternative` parameter here, as we did for the `deploy()` API?

Reply:
@trangevi `build_image()` was used as a step in the AzureML deployment procedure, so it makes sense to direct users towards the new deployment procedure, even if the new procedure doesn't involve a separate image building step. Can we specify the deployment plugin as an alternative?

Reply:
Done